spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -119,6 +119,11 @@
 	 * Connection timeout. Set it to zero to wait forever.
 	 */
 	private Duration connectionTimeout;
+	
+	/**
+	 * Channel RPC timeout.
+	 */
+	private Duration channelRpcTimeout;
```

Comment:
May be annotate with `@DurationUnit(ChronoUnit.SECONDS)` (see field `requestedHeartbeat`)
The default duration is infinite, which may lead to a freeze of the Spring Boot application startup in some situations
=> may be a sensible default of 1 min would be wise?

Reply:
@pierre-sion the default is already ms so need to change the unit. In the original PR, no default value was set (`null`) and the mapping only applies if a non null value is provided. The default of 10 minutes is preserved but I've made that more explicit in a polish commit.